src/components/mailFilter/ActionFileinto.vue

```diff
@@ -36,13 +37,28 @@ export default {
 		mailbox() {
 			return this.action.mailbox ?? undefined
 		},
-		mailboxes() {
-			const mailboxes = this.mainStore.getMailboxes(this.account.accountId)
+		mailboxOptions() {
+			const result = []
+
+			const topMailboxes = this.mainStore.getMailboxes(this.account.accountId)
 				.filter(mailbox => mailboxHasRights(mailbox, 'i'))
 
-			return mailboxes.map((mailbox) => {
-				return mailbox.displayName
-			})
+			for (const mailbox of topMailboxes) {
+				result.push({
+					value: mailbox,
+					label: mailbox.displayName,
+				})
+
+				const subMailboxes = this.mainStore.getSubMailboxes(mailbox.databaseId)
+				for (const subMailbox of subMailboxes) {
+					result.push({
+						value: subMailbox,
+						label: ` > ${subMailbox.displayName}`,
```

Comment:
What about mailboxes of level two or higher?

For example

* INBOX
* Archive
  * 2025
    * Inbox
    * Sent

How will that show?

Reply:
I didnt test that scenario, and i am not sure if its shown at all. Does that mean that we need to show all the levels here? I will check it in a bit

Reply:
we discussed on a call, we should show all available mailboxes

Reply:
i used mailboxinlinepicker,